### ISSUE TITLE
[BUG/ISSUE] Duplicate call to Init_and_Register SpeciesConc?

### ISSUE THREAD

**Issue by jimmielin** [geoschem/geos-chem#1021]
### Description of the problem
There is a piece of duplicate code calling `Init_and_Register` under `state_diag_mod.F90`

It appears to be harmless.

### GEOS-Chem version
Current code in `main` branch exhibits this issue.

### Replicating the issue:

Notice the two blocks of code here:

https://github.com/geoschem/geos-chem/blob/main/Headers/state_diag_mod.F90#L2151

https://github.com/geoschem/geos-chem/blob/main/Headers/state_diag_mod.F90#L2692

They appear to the exact same code:
```fortran
    !------------------------------------------------------------------------
    ! Species concentration diagnostic
    !------------------------------------------------------------------------
    diagId  = 'SpeciesConc'
    CALL Init_and_Register(                                                  &
         Input_Opt      = Input_Opt,                                         &
         State_Chm      = State_Chm,                                         &
         State_Diag     = State_Diag,                                        &
         State_Grid     = State_Grid,                                        &
         DiagList       = Diag_List,                                         &
         TaggedDiagList = TaggedDiag_List,                                   &
         Ptr2Data       = State_Diag%SpeciesConc,                            &
         archiveData    = State_Diag%Archive_SpeciesConc,                    &
         mapData        = State_Diag%Map_SpeciesConc,                        &
         diagId         = diagId,                                            &
         diagFlag       = 'S',                                               &
         RC             = RC                                                )

    IF ( RC /= GC_SUCCESS ) THEN
       errMsg = TRIM( errMsg_ir ) // TRIM( diagId )
       CALL GC_Error( errMsg, RC, thisLoc )
       RETURN
    ENDIF
```

They do not appear to be under any `#ifdef` so this looks to me as a duplicate.

I think one of them can be safely removed.

**Comment by yantosca** [comment]
I concur @jimmielin. That might have been leftover from a Git merge.  We can take the second instance out.

**Comment by yantosca** [comment]
This is now merged into the 13.4.0 development branch, so we can close out this issue.
